modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
+	err := e.db.View(func(tx *bolt.Tx) error {
+		var height types.BlockHeight
+		err := encoding.Unmarshal(tx.Bucket(bucketInternal).Get(internalBlockHeight), &height)
+		if err != nil {
+			return err
+		}
+		block, exists := e.cs.BlockAtHeight(height)
+		if !exists {
+			build.Critical(errors.New("latest explorer block doesnt exist in consensus set"))
+		}
+		bfBytes := tx.Bucket(bucketBlockFacts).Get(encoding.Marshal(block.ID()))
+		if bfBytes == nil {
+			build.Critical(errors.New("block facts for the latest explorer block dont exist"))
+		}
+		return encoding.Unmarshal(bfBytes, &bf)
```

Comment:
actually this function could just be:
```go
var height types.BlockHeight
err := e.db.View(dbGetInternal(internalBlockHeight, &height))
if err != nil {
    build.Critical(err)
}
return e.BlockFacts(height)
```
DRY and whatnot. The only downside is that this means you're doing two db transactions instead of one.